### PR TITLE
Add HTS Precompile Acceptance Tests for KYC methods

### LINKED ISSUE #407: Add htsPrecompile acceptance test support for kyc methods
### Problem

No htsPrecompile acceptance test support exists for kyc methods

### Solution

Add support for token kyc management flows

- `isKyc(address)`
- `tokenGrantKyc(address, address)`
- `tokenRevokeKyc(address, address)`

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by georgi-l95**
**Description**:
Add tests for HTS precompile:

- `isKyc(address)`
- `tokenGrantKyc(address, address)`
- `tokenRevokeKyc(address, address)`

**Related issue(s)**:

Fixes #407 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: packages/server/tests/acceptance/htsPrecompile.spec.ts [hunk 1]
```diff
@@ -517,55 +511,104 @@ describe('HTS Precompile Acceptance Tests', async function () {
...
-    expect(fixedFees[0].tokenId).to.equal(HTSTokenContractAddress);
-    expect(fixedFees[0].useHbarsForPayment).to.equal(false);
-    expect(fixedFees[0].useCurrentTokenForPayment).to.equal(false);
-
-    expect(fractionalFees[0].numerator).to.equal(4);
-    expect(fractionalFees[0].denominator).to.equal(5);
-    expect(fractionalFees[0].minimumAmount).to.equal(10);
-    expect(fractionalFees[0].maximumAmount).to.equal(30);
-    expect(fractionalFees[0].netOfTransfers).to.equal(false);
+    it('should be able to grant Kyc', async function() {
```
Comment (Nana-EC) [inline packages/server/tests/acceptance/htsPrecompile.spec.ts:539]: Are these set of KYC tests assuming they run sequentially?
Probably not good practice.
You could combine this into one test where you
- verify kyc is false to start
- grant kyc
- verify it's true
- verify token can now be sent
- revoke kyc
- verify it's false

At the acceptance test level this is acceptable :)

**Review by Nana-EC — CHANGES_REQUESTED** [review]
LG, please update to not rely on sequential run of tests, they should all be independent

File: packages/server/tests/acceptance/htsPrecompile.spec.ts (hunk 1 above)
Comment (georgi-l95) [reply packages/server/tests/acceptance/htsPrecompile.spec.ts:539]: Done. This flow was transferred to one test.

**Review by Nana-EC — APPROVED** [review]
LG

**Pull request merged**